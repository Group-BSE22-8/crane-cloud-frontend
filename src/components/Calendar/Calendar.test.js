/* eslint-disable no-undef */
import React from "react";
import { shallow } from "enzyme";
import Calendar from "./";

const CalendarProps = {
  prevMonth: jest.fn(),
  nextMonth: jest.fn(),
  daysInMonth: jest.fn(),
  renderDays: jest.fn(),
  handleSelected: jest.fn(),
  onChange: jest.fn(),
};


describe('TestContainer', () => {
  const CalendarWrapper = shallow(<Calendar {...CalendarProps} />);
  it('Test class constructor', () => {
  
    expect(CalendarWrapper.exists()).toBeTruthy();
    expect(CalendarWrapper).toHaveLength(1);
  });
  it("matchs the component snapshot", () => {
    expect(CalendarWrapper).toMatchSnapshot();
  });
  it("should match the snapshot", () => {

    CalendarWrapper.setProps(CalendarProps);
    expect(CalendarWrapper).toBeDefined();
    expect(CalendarWrapper).toHaveLength(1);
  });

});
