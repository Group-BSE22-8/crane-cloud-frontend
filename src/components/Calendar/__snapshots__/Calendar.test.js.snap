// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TestContainer matchs the component snapshot 1`] = `ShallowWrapper {}`;
